### DISCUSSION TITLE
Do I include the Microsoft.WindowsAppSdk.manifest file in my project?

### DISCUSSION THREAD

**Discussion by Zulu-Inuoe** [microsoft/WindowsAppSDK#3649] [Q&A]
I've installed the latest stable version `1.3.230502000` from NuGet, and as part of that, the file `Microsoft.WindowsAppSdk.manifest` was added automatically to my project.

Is this file necessary? Or should I delete and `.gitignore` it ?
Could somebody explain its purpose?

Thanks!

**Comment by DarranRowe** [comment]
Could you give more information as to where you see this file?
I don't see this file being added to any of my projects, all of them updated to 1.3.1 and they are varying between WinUI 3 projects to other projects that use the Windows App SDK.
